src/gui/systray.cpp

```diff
@@ -105,6 +105,13 @@ void Systray::create()
     }
     hideWindow();
     emit activated(QSystemTrayIcon::ActivationReason::Unknown);
+
+    const auto folderMap = FolderMan::instance()->map();
+    for (const auto *folder : folderMap) {
+        if(!folder->syncPaused()) {
+            _syncIsPaused = false;
```

Comment:
Similarly, my brain kicks in only now... *sigh*.

You can break the loop after that, no need to iterate further on the map.